packages/twenty-front/src/modules/object-record/record-picker/single-record-picker/components/SingleRecordPickerMenuItemsWithSearch.tsx

```diff
@@ -122,9 +122,7 @@ export const SingleRecordPickerMenuItemsWithSearch = ({
               onRecordSelected,
             }}
           />
-          {records.recordsToSelect.length > 0 && isDefined(onCreate) && (
-            <DropdownMenuSeparator />
-          )}
+          {isDefined(onCreate) && <DropdownMenuSeparator />}
           {isDefined(onCreate) && hasObjectUpdatePermissions && (
             <DropdownMenuItemsContainer scrollable={false}>
               {createNewButton}
```

Comment:
Would it make sense always to render the separator when the createNewButton is?

```suggestion
          {isDefined(onCreate) && hasObjectUpdatePermissions && (
            <>
              <DropdownMenuSeparator />
              <DropdownMenuItemsContainer scrollable={false}>
                {createNewButton}
```

If `hasObjectUpdatePermissions` is `false`, we would render a separator but no button below. I didn't test, but I assume it might cause issues.

Reply:
Yes I'll test

Reply:
Indeed, I fixed it